content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -316,5 +371,147 @@ following ways:
+
+It is important to note that only the products included in the original invoice can be part of the
+debit note. While you can make changes to the product's unit price or quantity, you cannot add new
+products to the debit note. The purpose of this document is to declare the amount that you want to
+add to the original invoice for the same or fewer products.
+
+To create a debit note from an invoice, go to the :guilabel:`⚙️ (gear)` icon on the left upper
+corner of the invoice and select :guilabel:`Debit Note`, then click on :guilabel:`Copy lines` and
+click :guilabel:`Create Debit Note`.
+
+.. image:: brazil/debit-note.png
+   :alt: Debit note creation in Odoo.
+
+.. note::
+   The required fields on the debit note are the same that apply for :ref:`customer invoices
+   <brazil/electronic-documents>`.
+
+Invoice cancellation
+~~~~~~~~~~~~~~~~~~~~
+
+It is possible to cancel an electronic invoice that was validated by the government.
+
+This can be done in Odoo by clicking :guilabel:`Request Cancel` and adding a cancellation
+:guilabel:`Reason` on the pop-up that appears. If you want to send this cancellation reason to your
+customer via email, you can do so by selecting the :guilabel:`E-mail` checkbox.
+
+Furthermore, it is important to check whether the electronic invoice is still within the
+cancellation deadline, which may vary according to the legislation of each state.
+
+.. image:: brazil/invoice-cancellation.png
+   :alt: Invoice cancellation reason in Odoo.
+
+Correction letter
+~~~~~~~~~~~~~~~~~
+
+It is possible to create a correction letter linked to an electronic invoice that was validated by
+the government.
+
+This can be done in Odoo by clicking :guilabel:`Correction Letter` and adding a correction
+:guilabel:`Reason` on the pop-up that appears. If you want to send this correction reason to your
+customer via email, you can do so by selecting the :guilabel:`E-mail` checkbox.
+
+.. image:: brazil/correction-letter.png
+   :alt: Correction letter reason in Odoo.
+
+Invalidate invoice number range
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A range of numbers within series that are assigned to sales journals can be invalidated with the
+government, if those sequences are not used and are not going to be used. To do so, navigate to the
+journal select the ::menuselection:`⚙️ (gear) icon --> Invalidate Number Range (BR)`. On the
+:guilabel:`Invalidate Number Range (BR)` wizard, add the :guilabel:`Initial Number` and
+:guilabel:`End Number` of the range that should be canceled, and state an invalidation
+:guilabel:`Reason`.
+
+.. image:: brazil/range-number-invalidation.png
+   :alt: Number range invalidation selection in Odoo.
+
+.. image:: brazil/range-number-invalidation-wizard.png
+   :alt: Number range invalidation wizard in Odoo.
+
+.. note::
+   The log of the numbers that were canceled and the XML file will be recorded in the chatter of the
+   journal.
+
+Electronic documents
```

Comment:
Hey @samueljlieber my bad here! This is wrong, it should be
```suggestion
Vendor Bills
```

Can you help me with this change?

All the rest looks good!
cc' @ren-odoo @fvz-odoo